src/meshapi/views/map.py

```diff
@@ -69,7 +69,7 @@ def get_queryset(self) -> List[Install]:  # type: ignore[override]
         # include the nns we haven't already covered via install num
         covered_nns = {install.install_number for install in all_installs}
         for node in (
-            Node.objects.filter(~Q(status=Node.NodeStatus.INACTIVE) & Q(installs__status__in=ALLOWED_INSTALL_STATUSES))
```

Comment:
It looks like `ALLOWED_INSTALL_STATUSES` is only mentioned here and on L216. Should we remove it there as well? And if so, should we remove the variable entirely?

Reply:
Good catch, I'll get rid of this entirely